examples/filesystem/Makefile

```diff
@@ -32,7 +32,8 @@ USEMODULE += mtd
 USEMODULE += vfs
 
 # Use a file system
-USEMODULE += littlefs
+# USEMODULE += littlefs
+USEMODULE += littlefs2
```

Comment:
Not that I mind, but this sneakily changes the default FS for the example :)

Reply:
I reverted it - thought it would be a good default since there is only one upstream littlefs - what we call littlefs2.